igvm/commands.py

```diff
@@ -432,6 +432,10 @@ def _reset_hypervisor():
                 _vm.start(transaction=transaction)
             _vm.reset_state()
 
+            # Add migration log entries to hypervisor and previous_hypervisor
+            hypervisor.hv_add_migration_log(_vm, '+')
```

Comment:
we are missing transaction rollback callbacks here. if for whatever reason something is failing or the vm commit below is failing, we will end up with invalid migration log entries on serveradmin

Reply:
Added the rollback callbacks.